app/src/main/java/io/gnosis/safe/ui/signing/owners/OwnerSelectionViewModel.kt

```diff
@@ -14,10 +15,11 @@ import javax.inject.Inject
 class OwnerSelectionViewModel
 @Inject constructor(
     private val derivator: MnemonicKeyAndAddressDerivator,
+    private val ownerKeyHandler: OwnerKeyHandler,
     appDispatchers: AppDispatchers
 ) : BaseStateViewModel<OwnerSelectionState>(appDispatchers) {
 
-    private var ownerIndex: Long = 1
+    private var ownerIndex: Long = 0
```

Comment:
Otherwise the wrong index is selected if the user doesn't tap on anything in the list.